app/Resources/DoctrineMigrations/Version20160627210338.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Ilios\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Replace AAMC resource type identifiers with the officially provided codes.
+ */
+class Version20160627210338 extends AbstractMigration
+{
+    /**
+     * @inheritdoc
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        // "virtual reality" has been dropped from the list, remove it.
+        $this->addSql('DELETE FROM aamc_resource_type WHERE resource_type_id = 24');
```

Comment:
Can we do this globally without checking?  Is it possible a campus has linked this to something already?